src/careamics/lvae_training/dataset/utils/data_utils.py

```diff
@@ -0,0 +1,197 @@
+"""
+Utility functions needed by dataloader & co.
+"""
+
+import os
+from typing import List
+
+import numpy as np
+from skimage.io import imread, imsave
+
+from careamics.lvae_training.dataset.configs.vae_data_config import (
+    DataSplitType,
+    DataType,
+)
+
+Multifile_Datasets = [
+    "Pavia3SeqData",
```

Comment:
Other things are okay. but this cannot work.  For example Pavia3SeqData has 9 different dataset. I do not see them anywhere. Similar things could be said about Nicola dataset. 
This classification between singlefile and multifile is also not correct. `OptiMEM100_014`, for example is a single file dataset. 

I would copy the rawdata loader codes for the different dataset. If this makes sense, you can have a submodule in ceramics which handles the data loading specifications for this work. this would allow you to not mix that much code into caremics. so, in careamics, you will have the basic code needed to run the different models. but the raw data loader part, which is tied to the paper, can live elsewhere.

Reply:
I added the parameter `load_data_fn`, where `get_train_val_data` functions can be passed from different experiments. 
I also manually went through all the python files for the dataset list that you gave, things we are missing now for the experiments to run:
1) `multifile_raw_dloader.py` file, used by 3 datasets, which I think should exist together with the code of the experiments
2)  dependencies: `czifile`, `nis2pyr`, `nd2`,